extensions/migadu/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Migadu Changelog
 
+## [Add Anonymous Identity Command] - 2024-10-13
```

Comment:
```suggestion
## [Add Anonymous Identity Command] - {PR_MERGE_DATE}
```

Reply:
Just a couple of items left:
- update the CHANGELOG date
- update the README to mention the new command (you could add a note mentioning one can use the new command for quick identity generating)

After that LGTM 🚀